Makefile

```diff
@@ -78,10 +78,17 @@ prepare-release:
 		$(MAKE) build-containers
 endif
 
+pypi-build:
+	echo ${KADALU_VERSION} > cli/VERSION
+	cd cli; rm -rf dist; python3 setup.py sdist;
```

Comment:
I am thinking of one more line for kadalu_quotad here. Do you want to do that as a separate patch? or change in this itself?